app/views/content/blog/top-tips-for-returning-teachers.md

```diff
@@ -85,4 +85,4 @@ Once you have made the decision to return to teaching what’s the best way of f
 
 Remember though that teacher recruitment goes in cycles, with most teaching roles being advertised from January onwards. Setting up a job alert will mean you don’t miss out on your perfect role.
 
-[Find out more about returning to teaching](/returning-to-teaching). If you’re a qualified maths, physics or languages teacher, [get your Return to Teaching Adviser now](https://adviser-getintoteaching.education.gov.uk/).
+[Find out more about returning to teaching](/returning-to-teaching). If you’re a qualified maths, physics, chemistry, computing or languages teacher, [get your Return to Teaching Adviser now](https://adviser-getintoteaching.education.gov.uk/).
```

Comment:
Again, I wonder if we should change this to say 'want to teach maths..... and have qualified teacher status (QTS)

(NB I know this was the original wording and not your changes!)